qemu/tests/cfg/win_virtio_driver_install_from_update.cfg

```diff
@@ -0,0 +1,128 @@
+    chk_timeout = 240
+    variants:
+        - with_netkvm:
+            driver_name = netkvm
+            device_name = "Red Hat VirtIO Ethernet Adapter"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1000" "PCI\VEN_1AF4&DEV_1041"'
+            nics += " nic2"
+            nic_model_nic1 = rtl8139
+            nic_model_nic2 = virtio
+        - with_viorng:
+            driver_name = viorng
+            device_name = "VirtIO RNG Device"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1005" "PCI\VEN_1AF4&DEV_1044"'
+            no_virtio_rng:
+                virtio_rngs += " rng0"
+                backend_rng0 = rng-builtin
+                backend_type = builtin
+        - with_viostor:
+            driver_name = viostor
+            device_name = "Red Hat VirtIO SCSI controller"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1001" "PCI\VEN_1AF4&DEV_1042"'
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = virtio
+            force_create_image_stg = yes
+            remove_image_stg = yes
+        - with_vioscsi:
+            driver_name = vioscsi
+            device_name = "Red Hat VirtIO SCSI pass-through controller"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1004" "PCI\VEN_1AF4&DEV_1048"'
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = scsi-hd
+            force_create_image_stg = yes
+            remove_image_stg = yes
+        - with_vioserial:
+            driver_name = vioser
+            device_name = "VirtIO Serial Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1003" "PCI\VEN_1AF4&DEV_1043"'
+            serials += " vs"
+            serial_type_vs = virtserialport
+        - with_balloon:
+            driver_name = balloon
+            device_name = "VirtIO Balloon Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1002" "PCI\VEN_1AF4&DEV_1045"'
+            balloon = balloon0
+            balloon_dev_devid = balloon0
+            balloon_dev_add_bus = yes
+        - with_pvpanic:
+            no Host_RHEL.m6
+            driver_name = pvpanic
+            device_name = "QEMU PVPanic Device"
+            device_hwid = '"ACPI\QEMU0001"'
+        - with_vioinput:
+            no Host_RHEL.m6 Host_RHEL.m7.u0 Host_RHEL.m7.u1 Host_RHEL.m7.u2 Host_RHEL.m7.u3
+            no Win2008..sp2
+            driver_name = vioinput
+            device_name = "VirtIO Input Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
+            inputs = input1
+            input_dev_bus_type_input1 = virtio
+            variants:
+                - device_mouse:
+                    input_dev_type_input1 = mouse
+                - device_keyboard:
+                    input_dev_type_input1 = keyboard
+                - device_tablet:
+                    input_dev_type_input1 = tablet
+        - with_viofs:
```

Comment:
These definitions about different devices, are testing same things here: https://github.com/autotest/tp-qemu/pull/2482/files#diff-d3306f225c3be84da056919b5fd5b05589ebe681050bf5fb72f0a276338678d3R67, right? If so, I  suppose it would be better to organize this case together within that one, I mean in the same cfg file.

Reply:
The devices are same but in this patch I should test it one by one and for patch https://github.com/autotest/tp-qemu/pull/2482 I need to boot all devices together, so I had better keep this cfg file .

Reply:
@vivianQizhu  could you please help to review this patch when you free, thanks